### PR TITLE
Observability - Initial Doc

### PR DESCRIPTION
**Pull request by mrhoads**
This doc does *not* contain screenshots because of current issues with the E2E environment.   I'm submitting it to get early feedback on the rest of the content.

### CODE REVIEW

File: docs/azure_jumpstart_ag/contoso_hypermarket/observability/_index.md [hunk 1]
```diff
@@ -5,4 +5,48 @@ title: Observability
...
 ---
 
-# Observability
\ No newline at end of file
+# Infrastructure observability Arc-enabled Kubernetes
+
+## Overview
+
+
+Infrastructure observability is essential for the success of Contoso Hypermarket's cloud-to-edge strategy. By using various tools for infrastructure observability, Contoso Hypermarket is able to monitor and visualize the health of their Kubernetes and Arc-enabled Kubernetes environments. This allows them to proactively oversee the health and performance of their infrastructure, identify potential issues, and make data-driven decisions to optimize their operations. With infrastructure observability, Contoso can ensure that their cloud and edge infrastructure remains reliable, efficient, and resilient, enabling them to deliver outstanding customer experiences.
```
Comment (dkirby-ms) [inline docs/azure_jumpstart_ag/contoso_hypermarket/observability/_index.md:13]: last sentence drags on a bit, perhaps remove.

File: docs/azure_jumpstart_ag/contoso_hypermarket/observability/_index.md [hunk 2]
```diff
@@ -5,4 +5,48 @@ title: Observability
...
+
+Prometheus is an efficient open-source monitoring system that collects and stores metrics from various sources in real-time. It offers a flexible query language for analyzing the collected metrics and provides robust alerting capabilities. Meanwhile, Grafana is a powerful open-source data visualization and analytics platform that allows users to create interactive and customizable dashboards to visualize the collected metrics in real-time.
+
+By using Prometheus and Grafana for infrastructure observability, Contoso benefits in three key ways. First, Prometheus's efficient data collection ensures that Contoso can monitor critical performance indicators and resource utilization in real-time. Secondly, Grafana offers a user-friendly interface for visualizing the collected metrics, enabling Contoso to create interactive and customizable dashboards. These dashboards provide valuable insights into their infrastructure’s health and performance, facilitate trend identification, and support informed decision-making. Third, the combination of Prometheus and Grafana aids in troubleshooting and root cause analysis.
```
Comment (dkirby-ms) [inline docs/azure_jumpstart_ag/contoso_hypermarket/observability/_index.md:17]: suggest making the three benefits into bullets

**Review by dkirby-ms — CHANGES_REQUESTED** [review]
Minor feedback, overalll on the right path. When finished with screenshots will give another pass.

**Comment by mrhoads** [comment]
closing this as OBE.  New PR will be submitted for infra observability

**Pull request closed without merging**